### PR TITLE
Fix #1298: Add cancel button to submission update

### LINKED ISSUE #1298: Add cancel button in update submission metadata
## Current Behaviour
The update submission is a modal form which has only submit button to submit and it can be disabled using the `Esc` key. Please see the screenshot.
cancel
## Expected Behaviour
We need to add a cancel button near the submit button so as to make the disable feature more user-friendly.
Please let us know if something is unclear.

### PR DESCRIPTION
**Pull request by tlugger**
This should close #1298. Adding a cancel button to the update submission modal. Please let me know if this needs any changes!

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@tlugger Can you please paste the screenshot ?

**Comment by tlugger** [comment]
screen shot 2017-10-02 at 8 34 26 am

**Comment by RishabhJain2018** [comment]
Hey, Can you please reorder the buttons and also make the UI according to the screenshot below :)
cancel2

**Comment by tlugger** [comment]
Sure thing! I should have those updates up tonight.

**Comment by tlugger** [comment]
@RishabhJain2018 how is this?

screen shot 2017-10-02 at 11 58 07 am

File: frontend/src/views/web/challenge/update-submission-metadata.html [hunk 1]
```diff
@@ -24,9 +24,10 @@
                         <span class="form-icon form-icon-dark"><i class="fa fa-pencil"></i></span>
                         <label for="publicationUrl">Publication Url</label>
                     </div>
-                    <div class="align-left reg-control">
-                        <button class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" type="submit" value="Submit">Submit </button>
-                    </div>
+                    <ul class="inline-list">
+                        <li><a class="dark-link" type="button" ng-click="challenge.updateSubmissionMetaData(false)"><strong>Cancel </strong></a></li>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/update-submission-metadata.html]: Can you please add a class `pointer` to it.

**Comment by tlugger** [comment]
Added

**Comment by deshraj** [comment]
Thanks for the fix @tlugger. Merging this.

**Pull request merged**